### PR TITLE
 plasma-hud: init at 19.10.1 

### PR DESCRIPTION
**Pull request by pasqui23**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/misc/plasma-hud/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{lib, python3, fetchFromGitHub, rofi, gobject-introspection}:
+with python3.pkgs;
+buildPythonApplication rec{
```
Comment (SuperSandro2000) [inline pkgs/misc/plasma-hud/default.nix:3]:
```suggestion
{ lib, python3, fetchFromGitHub, rofi, gobject-introspection }:

buildPythonApplication rec{
```

Please apply the with where necessary.

File: pkgs/misc/plasma-hud/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  patchPhase = ''
+    sed -i "s:/usr/lib/plasma-hud:$out/bin:" etc/xdg/autostart/plasma-hud.desktop
+  '';
+  installPhase = ''
+    patchShebangs
+    mkdir -p $out/{bin,etc/xdg/autostart}
+    cp -r {$src/usr/lib/plasma-hud,$out/bin}/plasma-hud
+    cp -r {$src,$out}/etc
+  '';
+  meta = with lib;{
```
Comment (SuperSandro2000) [inline pkgs/misc/plasma-hud/default.nix:32]:
```suggestion
    sed -i "s:/usr/lib/plasma-hud:$out/bin:" etc/xdg/autostart/plasma-hud.desktop
  '';

  installPhase = ''
    patchShebangs
    mkdir -p $out/bin $out/etc/xdg/autostart
    cp -r $src/usr/lib/plasma-hud/plasma-hud $out/bin/plasma-hud
    cp -r $src/etc $out/etc
  '';

  meta = with lib;{
```

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -13537,6 +13537,9 @@ in
 
   pyotherside = libsForQt5.callPackage ../development/libraries/pyotherside {};
 
+  plasma-hud = callPackage ../misc/plasma-hud/default.nix { };
+
+
   re2 = callPackage ../development/libraries/re2 { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:13543]:
```suggestion

  re2 = callPackage ../development/libraries/re2 { };
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/misc/plasma-hud/default.nix [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    license = licenses.gpl2;
```
Comment (SuperSandro2000) [inline pkgs/misc/plasma-hud/default.nix:33]: gpl2Only or gpl2Plus?

File: pkgs/misc/plasma-hud/default.nix [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    homepage = "https://github.com/Zren/plasma-hud";
+    platforms = platforms.unix;
+    description = "Run menubar commands, much like the Unity 7 Heads-Up Display (HUD)";
```
Comment (SuperSandro2000) [inline pkgs/misc/plasma-hud/default.nix:36]: Missing maintainers.

File: pkgs/misc/plasma-hud/default.nix [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    xlib
+    pygobject3
+    gobject-introspection
+  ];
+  format = "other";
+  patchPhase = ''
+    sed -i "s:/usr/lib/plasma-hud:$out/bin:" etc/xdg/autostart/plasma-hud.desktop
+  '';
+  installPhase = ''
+    patchShebangs
```
Comment (SuperSandro2000) [inline pkgs/misc/plasma-hud/default.nix:27]: Please be more specific.

Comment (pasqui23) [reply pkgs/misc/plasma-hud/default.nix:27]: What do you mean?

Comment (SuperSandro2000) [reply pkgs/misc/plasma-hud/default.nix:27]: Which files should be patched? Please list them instead of trying to patch every file.

**Comment by pasqui23** [comment]
ok I've addressed @SuperSandro2000 's concerns

File: pkgs/misc/plasma-hud/default.nix [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  propagatedBuildInputs = with python3.pkgs; [
+    rofi
+    dbus-python
+    setproctitle
+    xlib
+    pygobject3
+    gobject-introspection
+  ];
+  format = "other";
+  patchPhase = ''
```
Comment (SuperSandro2000) [inline pkgs/misc/plasma-hud/default.nix:23]:
```suggestion
  postPatch = ''
```

File: pkgs/top-level/all-packages.nix [hunk 8]
```diff
@@ -13537,6 +13537,8 @@ in
 
   pyotherside = libsForQt5.callPackage ../development/libraries/pyotherside {};
 
+  plasma-hud = callPackage ../misc/plasma-hud/default.nix { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:13540]:
```suggestion
  plasma-hud = callPackage ../misc/plasma-hud { };
```

**Pull request merged**